packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -407,7 +411,7 @@ const CreateAdForm = () => {
                                                 className='p2p-my-ads__form-button'
                                                 secondary
                                                 large
-                                                onClick={() => my_ads_store.setShowAdForm(false)}
+                                                onClick={() => onCleanup()}
```

Comment:
can it be like this?
```suggestion
                                                onClick={onCleanup}
```